src/sentry/conf/server.py

```diff
@@ -132,6 +132,10 @@ def env(key, default="", type=None):
     "ulimits": [{"name": "nofile", "soft": 262144, "hard": 262144}],
     "environment": {"MAX_MEMORY_USAGE_RATIO": "0.3"},
 }
+CLICKHOUSE_CONFIG_FILE = (
+    "dist_config.xml" if SENTRY_DISTRIBUTED_CLICKHOUSE_TABLES else "loc_config.xml"
+)
+CLICKHOUSE_VOLUME = "clickhouse_dist" if SENTRY_DISTRIBUTED_CLICKHOUSE_TABLES else "clickhouse"
```

Comment:
We shouldn't make these variables that the user can override. Since they are only used in one place, can you just move them inline?

Reply:
Same for `_common_clickhouse_settings` above. Since we are only using it in one place now, please inline that too.

Reply:
Good catch, I made both inline.